gap/projective/almostsimple.gi

```diff
@@ -1429,3 +1430,114 @@ FindHomMethodsProjective.SporadicsByOrders := function(ri,G)
   od;
   return false; # FIXME: false = NeverApplicable here really correct?
 end;
+
+# Data for the function NameSporadic
+BindGlobal("NameSporadicData", MakeImmutable([
+    rec(name := "M11",
+        maximalOrdersOverset := [5, 6, 8, 11],
+        length := 4),
```

Comment:
What's `length` for? It doesn't seem to be used?

Reply:
Oh, it's a mistake that that isn't used. For M11, M12, M22 the maximalOrders set has to be equal to maximalOrdersOverset. I'm solving this by doing 
`maximalOrdersSubset := ~.maximalOrdersOverset),` for those groups